docs/developer_reference/getting_started.md

```diff
@@ -2,56 +2,73 @@
-    -p 8443:8334 \
-    -p 8888:8080 \
-    -d \
-    bcgsc/pori-auth:latest
+mkdir -p databases/{postgres,orientdb}/{backup,data}
 ```
 
-You should now be able to view the browser-based administrative console by visiting [http://localhost:8888](http://localhost:8888) in your browser.
+You should also create a new directory for storing the public key from keycloak. This key will be downloaded and store so that it was be used in checking incoming tokens by the GraphKB and IPR APIs. If this directory already exists you should delete and remake it.
 
-### Download the Public Key File
+```bash
+mkdir keys
+```
+
+Now you are ready to start up with the dev compose yml
 
-After the container is started you can go to the admin console GUI to add a users and download the realm's public key file. This must be done prior to starting the other containers.
+```bash
+docker-compose -f docker-compose.dev.yml up -d
+```
 
-You can do this via the GUI as described in the main [install instructions](../install.md) or via a script using the keycloak REST API.
+It will take a minute or two for all of the servers to start. You can check how they look with docker
 
-```bash title="kc_setup_keyfile.sh"
---8<-- "./kc_setup_keyfile.sh"
+```bash
+docker ps
 ```
 
-### Create the Default Users
+If any of them show "(health: starting)" then they are not ready yet.
+
+### Viewing Log Files
 
-Next, create the users as specified in the main [install instructions](https://bcgsc.github.io/pori/install) or via the script below.
+Sometimes you will need to check the logs from the various servers, this can be done with the docker logs command. First find the container ID (or name) by listing all the running containers with `docker ps` and then run the following
 
-```bash title="kc_setup_default_users.sh"
---8<-- "./kc_setup_default_users.sh"
+```bash
+docker logs <CONTAINER ID>
 ```
 
-## Run docker-compose
+### Loading Data into GraphKB
 
-Once keycloak is set up you will need to create some directories for storing database data (so that is persists when you stop and restart your docker containers)
+If you are running the GraphKB loader via its docker container you will need to tell it to use the host network so that it is able to find the GraphKB API.
+
+Here is an example of running the GraphKB Loader on the vocabulary terms using the docker container and the docker-compose setup described above.
+
+First download the vocabulary terms data
 
 ```bash
-mkdir -p databases/{postgres,orientdb}/{backup,data}
+wget https://raw.githubusercontent.com/bcgsc/pori_graphkb_loader/develop/data/vocab.json
 ```
 
-Now you are ready to start up with the dev compose yml
+Then you can load these terms using the ontology file loader
 
 ```bash
-docker-compose -f demo/docker-compose.dev.yml up -d
+docker run --net host bcgsc/pori-graphkb-loader:latest \
+    -u graphkb_importer \
+    -p secret \
+    -g http://localhost:8888/api \
+    file \
+    ontology \
+    vocab.json
```

Comment:
Recap of discussion elsewhere - on testing this out it turned out a --mount arg was needed. Absolute path for vocab.json was needed (when I ran it) for source and dst components of the --mount arg as well as the filename after 'ontology'.